x/bank/spec/02_keepers.md

```diff
@@ -4,9 +4,27 @@ order: 2
 
 # Keepers
 
-The bank module provides three different exported keeper interfaces which can be passed to other modules which need to read or update account balances. Modules should use the least-permissive interface which provides the functionality they require.
+The bank module provides three different exported keeper interfaces which can be
+passed to other modules which need to read or update account balances. Modules
+should use the least-permissive interface which provides the functionality they
+require.
```

Comment:
```suggestion
The bank module provides these exported keeper interfaces that can be
passed to other modules that read or update account balances. Modules
should use the least-permissive interface that provides the functionality they
require.
```

Reply:
now we have more than three, best practice is to not count them "three exported keeper interfaces" becomes "these..."